keyboards/doio/kb3x/rules.mk

```diff
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes

```

Comment:
```suggestion

```
Guess my English wasn't up to par from the last suggestion. I should've meant that this should go into `kb3x/keymaps/default/rules.mk`.

Reply:
@adophoxia  I've got it. thank you.